inc/class-wpseo-rank.php

```diff
@@ -155,9 +155,9 @@ public function get_drop_down_label() {
 	 */
 	public function get_drop_down_readability_labels() {
 		$labels = array(
-			self::BAD      => __( 'Readability: Needs improvement', 'wordpress-seo' ),
-			self::OK       => __( 'Readability: OK', 'wordpress-seo' ),
-			self::GOOD     => __( 'Readability: Good', 'wordpress-seo' ),
+			self::BAD      => sprintf( __( 'Readability: %s', 'wordpress-seo' ), __( 'Needs improvement', 'wordpress-seo' ) ),
```

Comment:
Please add translator comments about the %s like in the files above.

Reply:
Done.